src/System.Linq/tests/EnumerableTests.cs

```diff
@@ -0,0 +1,131 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using Xunit;
+
+namespace System.Linq.Tests
+{
+    public partial class EnumerableTests
+    {
+        [Fact]
+        public void Average()
+        {
+            var array = Enumerable.Range(1, 10).ToArray();
+            var min = array.Min();
+            Assert.Equal(min, 1);
+        }
```

Comment:
Presumably there will be other tests for Min, e.g. with an empty sequence, with a sequence with only one value, with negative values, with an enumerator that throws an exception, etc.  Do you plan to do all of those in a single Min() test, or do you plan to split them across multiple Min_SomeSpecificTest() tests?  I don't have a strong preference either way, but if your plan is the latter, you probably want to start with that naming scheme now.